src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -36,25 +36,20 @@ With [our Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricin
 </figcaption>
 
 ## Requirements [#requirements]
 
-Being able to configure data retention requires Pro or Enterprise edition with billing administrator rights. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting retention on their own. If you're a Pro or Enterprise organization on a billing administrator account, and you can't edit retention, talk to your New Relic account representative.
+To upgrade your retention plan, or edit your effective retention values when you have more than one account in your organization, you need to be on the Pro or Enterprise edition. With these editions, you can designate users who are responsible for managing the retention plan or the effective amount of time data will be retained. Whether you have one user handle both tasks or assign separate users for each, you enable access differently:
 
-Note that our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus) provides up to 
-an additional 90 days of retention.
-
-Permission-related requirements for viewing and editing retention:
-* [Newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models): 
-  * Setting a contract with longer data retention for your entire organization requires the [<DoNotTranslate>**Billing**</DoNotTranslate> administration setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings)
-  * Adjusting retention within the bounds of a contract for specific types of data or for a specific account requires the [<DoNotTranslate>**Insights event retention**</DoNotTranslate> permission](/docs/accounts/accounts-billing/new-relic-one-user-management/user-permissions/#data-retention) 
-* [Original user model](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model): requires the Owner role or Data Retention Manager Add-On role.
-
-## The data retention UI [#find-ui]
-
-The <DoNotTranslate>**Data retention**</DoNotTranslate> UI page is located in the [Data management hub](https://one.newrelic.com/data-management-hub). 
+* **Upgrade your retention plan (Max per contract)**: Your **Max per contract** column specifies the maximum number you can configure for the **Effective retention** of a data type. This maximum allowance always applies to all accounts in your organization if you have more than one account. To increase your **Max per contract**, you need a user who is in a group that has the **Billing** administrative setting enabled for it. This administrative setting applies to all users in that group. Those users will have a **Billing User** role assigned to them, but this method is the only way to assign that specific role. Only these users will see an **Upgrade plan** button in the top right of the **Data retention** UI.
```

Comment:
@rhetoric101 This is pretty dense, technical info. I think it'd be good to look for places where we can simplify things a bit:

"To increase your **Max per contract**, you need a user who is in a group that has the **Billing** administrative setting enabled for it." 

could be

"To increase your **Max per contract**, you need a user with the **Billing** administrative setting enabled."

In general, I think this stuff is pretty dense. For example, we're trying to convey info about settings and also about groups as an administrative user structure. Can we separate these things a little?

For example, this ONE bullet is communicating all of this:
1. The max per contract column is the max number for effective retention by data type.
2. This setting applies to all your accounts (a universal or global setting)
3. To change this, you need a user who's in a group.
4. That group needs to have the "Billing" permissions/setting enabled
5. This setting assigns a user role called "Billing User"
6. This setting is the only way to assign that role.
7. Oh, by the way, they're also the only ones who see this one button in the UI.

See what I mean? :)

Reply:
@paperclypse I've done a few more revisions with the help of our SME. I'm closing your comments here since we've made a set of wholesale changes.